lib/practice_exercises.rb

```diff
@@ -1,13 +1,34 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) where n is the sorted array
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  counter = 0
+  list.length.times do |num|
+    if list[counter] == list[counter + 1]
+      list.delete_at(counter + 1)
```

Comment:
Since `delete_at` shifts all subsequent element left one index, it has a runtime of O(n).  Using in your loop here gives you a method time complexity of O(n<sup>2</sup>)